README.md

```diff
@@ -48,6 +48,12 @@ To use the script, you will need to follow these steps:
 
 All optional values above can also be specified on the command line.
 
+To run the script inside a docker container:
+
+```
+docker-compose run babyagi
```

Comment:
Just a question: I know, that you can pass command line arguments when running `docker run`. Does it work with `docker-compose` the same way?

Reply:
yeah, there's various ways to do this with compose: https://docs.docker.com/compose/environment-variables/set-environment-variables/